extensions/eo/README.md

```diff
@@ -22,20 +22,28 @@ For defining view geometry of data, it is strongly recommended to use the [`view
   - [Landsat 8 with bands in Item Asset Definition and Collection Summaries](../item-assets/examples/example-landsat8.json)
 - [JSON Schema](json-schema/schema.json)
 
-## Item fields
+## Item Asset fields
```

Comment:
It wasn't clear from this text into which entities these fields were supposed to go.

Reply:
We are generally allowing any fields from Item properties in Item Assets, so no need to specify this explicitly. Otherwise you'd need to duplicate the description for all the other fields as well, e.g. eo:cloud_cover and all other extensions.

https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#asset-object says: 
> Beyond to the fields above, any of the additional fields may be added to the assets. But this is recommended only in special cases, see Additional Fields for Assets) for more information.

So I'd prefer to remove this duplication. What we better add is a general hint that eo:bands might be better suited in assets (refering back to what I quoted abvoe) in certain cases.

Reply:
Thanks for clarifying. I'll update this now that I understand what that means.

Reply:
Maybe we need to improve how we describe the general behavior.

Reply:
I'll also look at clarifying the general behavior description.